packages/jellyfish-api-core/__tests__/category/icxorderbook/icx_setup.ts

```diff
@@ -107,4 +107,17 @@ export class ICXSetup {
     expect(result.ICX_TAKERFEE_PER_BTC as number).toStrictEqual(fee)
     ICX_TAKERFEE_PER_BTC = result.ICX_TAKERFEE_PER_BTC as number
   }
+
+  async closeAllOpenOffers (): Promise<void> {
+    const orders = await this.container.call('icx_listorders', [])
+    for (const orderTx of Object.keys(orders).splice(1)) {
+      const offers = await this.container.call('icx_listorders', [{ orderTx: orderTx }])
+      for (const offerTx of Object.keys(offers).splice(1)) {
+        if (offers[offerTx].status === 'OPEN') {
+          await this.container.call('icx_closeoffer', [offerTx])
+          await this.container.generate(1)
+        }
+      }
+    }
```

Comment:
```suggestion
        }
      }
    }
     await this.container.generate(1)
```

You can try doing it like this,  only do block gen after sending all close offer